docs/tutorials/eda/eda-auto-analyze-interaction.md

```diff
@@ -0,0 +1,88 @@
+# Feature Interaction Charting
+
+This tool is made for quick interactions visualization between variables in a dataset. User can specify the variables to
+be plotted on the x, y and hue (color) parameters. The tool automatically picks chart type to render based on the
+detected variable types and renders 1/2/3-way interactions.
+
+This feature can be useful in exploring patterns, trends, and outliers and potentially identify good predictors for the
+task.
+
+## Using Interaction Charts for Missing Values Filling
+
+Let's load titanic dataset:
+
+```{.python .input}
+import pandas as pd
+
+df_train = pd.read_csv('https://autogluon.s3.amazonaws.com/datasets/titanic/train.csv')
+df_test = pd.read_csv('https://autogluon.s3.amazonaws.com/datasets/titanic/test.csv')
+target_col = 'Survived'
+```
+
+Next we will look at missing data in the variable:
+
+```{.python .input}
+import autogluon.eda.auto as auto
+
+auto.missing_values_analysis(train_data=df_train)
+```
+
+It looks like there are only two null values in the `Embarked` feature. Let's see what are those two null values:
+
+```{.python .input}
+df_train[df_train.Embarked.isna()]
+```
+
+We may be able to fill these by looking at other independent variables. Both passengers paid a `Fare` of `$80`, are
+of `Pclass` `1` and `female` `Sex`. Let's see how the `Fare` is distributed among all `Pclass` and `Embarked` feature
+values:
+
+```{.python .input}
+auto.analyze_interaction(train_data=df_train, x='Embarked', y='Fare', hue='Pclass')
+```
+
+The average `Fare` closest to `$80` are in the `C` `Embarked` values where `Pclass` is `1`. Let's fill in the missing
+values as `C`.
+
+## Using Interaction Charts To Learn Information About the Data
+
+```{.python .input}
+auto.analyze_interaction(x='Pclass', y='Survived', train_data=df_train, test_data=df_test)
+```
+
+It looks like `63%` first class passengers survived; `48%` second class and only `24%` third class passenger survived.
```

Comment:
Here's few nits: "It looks like `63%` of first class passengers survived, while `48%` of second class and only `24%` of third class passengers survived."

Reply:
fixed